lib/packer.js

```diff
@@ -37,7 +38,7 @@ Packer.prototype.pack = function(data, width, height, gamma) {
   var filteredData = filter(data, width, height, this._options);
 
   // compress it
-  var deflate = zlib.createDeflate({
+  var deflate = options.deflateFactory({
```

Comment:
nice.  i think options here needs to be this._options - thats why the build fails.